std_msgs/Header header

float64 lat
float64 lon
float32 yaw
float32 north_velocity
float32 east_velocity

